function/requirements.txt

```diff
@@ -0,0 +1,3 @@
+fastavro
```

Comment:
I think we should be explicit about versions when going to production. I ran into this with Calhoun where I had to compare timestamps of deploys and library releases to figure out what version was running when. Made it difficult to check a theory that a version change was responsible for a change in behavior.

Reply:
Strong +1; this ate us on Leo too. Running `pip freeze` in an install where this works will print out versions of everything you're using which you can copy-paste into requirements.txt.